config/config.json

```diff
@@ -1,14 +1,14 @@
 {
   "development": {
     "username": "root",
-    "password": "password",
+    "password": "Mypassword1!",
     "database": "cinder_db",
     "host": "127.0.0.1",
     "dialect": "mysql"
   },
   "test": {
     "username": "root",
-    "password": "password",
```

Comment:
can we still have password in here? because everybody has the same password in workbench i believe